vmdb/app/controllers/ops_controller/diagnostics.rb

```diff
@@ -1062,6 +995,21 @@ def diagnostics_set_form_vars
     end
   end
 
+  def build_backup_schedule_options_for_select
+    @backup_schedules = {}
+    database_details
+    miq_schedules = MiqSchedule.all(:conditions => "towhat = 'DatabaseBackup' And adhoc is NULL")
```

Comment:
I think this should be: `MiqSchedule.where(:towhat => 'DatabaseBackup', :adhoc => nil)`.  We should avoid the Rails 2.3 style finders.